src/core/audio/AudioSdl.cpp

```diff
@@ -107,10 +161,24 @@ void AudioSdl::stopProcessing()
 {
 	if( SDL_GetAudioStatus() == SDL_AUDIO_PLAYING )
```

Comment:
@Reflexe LMMS crashes when rendering with SDL2 + this PR because this function doesn't stop playback correctly. You need to use `SDL_GetAudioDeviceStatus(m_outputDevice)` for SDL2 to fix this, since you used `SDL_OpenAudioDevice()`.